### PR TITLE
Using unit abbreviations instead of full name when viewing a recipe

### PR DESCRIPTION
**Pull request by maxmikk**
When viewing a recipe, the full unit name used to be displayed. This change will use the abbreviation instead.

### CODE REVIEW

File: View/Recipes/view.ctp [hunk 1]
```diff
@@ -220,7 +220,7 @@ if (isset($recipe['Review'])) {
                             if (isset($scale)) $quantity *= $scale;
                             $quantity = $this->Fraction->toFraction($quantity);
                             $unit = $related['Related']['IngredientMapping'][$i]['Unit']['name'];
-                            $ingredientName = $related['Related']['IngredientMapping'][$i]['Ingredient']['name'];
+                            $ingredientName = $related['Related']['IngredientMapping'][$i]['Ingredient']['name'];
```
Comment (nazgul26) [inline View/Recipes/view.ctp:223]: Not quite a revert.  You have a trailing space added.  I know small, but please remove.

File: Controller/RecipesController.php [hunk 2]
```diff
@@ -95,7 +95,7 @@ public function view($id = null, $servings=null) {
                             'fields' => array('name')
                         ),
                         'Unit' => array(
-                            'fields' => array('name')
+                            'fields' => array('name', 'abbreviation') //2017-01-23 - added 'abbreviation' to be able to view those in the recipe view.
```
Comment (nazgul26) [inline Controller/RecipesController.php]: Remove the comments.  We have source control for comments.  I know you mean well, but if every change was dated in code it quickly becomes unreadable overall.  Same goes for the other comments/dates.

**Review by nazgul26 — CHANGES_REQUESTED** [review]
Thanks for jumping in.  I know it is lot to get started with Git and all this.  Congrats on submitted the pull request.  Please see the review comments.

**Comment by maxmikk** [comment]
Comments removed.

**Pull request merged**